docs/scripts/generate_gallery_text.py

```diff
@@ -88,6 +88,7 @@ def regenerate_gallery():
             item.pop("repo", None)
 
             # add the new gallery item to the gallery file
+            item["path_image"] = Path(f"../_static/gallery/{item.pop('id')}.png")
```

Comment:
```suggestion
            item["path_image"] = Path("..")/"_static"/"gallery"/f"{item.pop('id')}.png"
```

I think you are forced to use the real split operator instead of integrating them in the path string